admin/src/App.tsx

```diff
@@ -1,10 +1,13 @@
 import { Admin, Resource } from "react-admin";
 import { dataProvider } from "./dataProvider";
+import { GameList } from "./Game/List";
```

Comment:
you could import like import { UserCreate, UserEdit, UserList } from "./User";

Reply:
I will check after the demo